en/tutorials/video_streaming.md

```diff
@@ -1,199 +1,17 @@
-SSID is the hotspot's name which is broadcast to other devices, channel is what frequency the hotspot will run on, wpa_passphrase is the password for the wireless network.
-For many more options see the file `/usr/share/doc/hostapd/examples/hostapd.conf.gz`.
-Look for a channel that is not in use in the area.
-You can use tools such as *wavemon* for that.
-
-Edit the file `/etc/default/hostapd` and change the line:
-
-```
-#DAEMON_CONF=""
-```
-to:
-```
-DAEMON_CONF="/etc/hostapd/hostapd.conf"
-```
-Your Onboard Computer should now be hosting a wireless hotspot.
-To get the hotspot to start on boot, run these additional commands: 
-
-```
-sudo update-rc.d hostapd enable
-sudo update-rc.d udhcpd enable
-```
-
-This is enough to have the Onboard Computer present itself as an Access Point and allow your ground station to connect.
-If you truly want to make it work as a real Access Point (routing the WiFi traffic to the Onboard Computer’s Ethernet connection), we need to configure the routing and network address translation (NAT).
-Enable IP forwarding in the kernel: 
-
-```sh
-sudo sh -c "echo 1 > /proc/sys/net/ipv4/ip_forward"
-```
+QGroundControl uses GStreamer 1.14.4 and a stripped down version of QtGstreamer to support UDP RTP and RSTP video streaming.
 
-To enable NAT in the kernel, run the following commands:
+To support streaming use cases you will need to install GStreamer development packages on both your companion computer and on the system running QGroundControl.
 
-```sh
-sudo iptables -t nat -A POSTROUTING -o eth0 -j MASQUERADE
-sudo iptables -A FORWARD -i eth0 -o wlan0 -m state --state RELATED,ESTABLISHED -j ACCEPT
-sudo iptables -A FORWARD -i wlan0 -o eth0 -j ACCEPT
-```
+> **Note** You can't video stream directly from a camera connected to PX4.
+  You will need to stream from a companion computer connected to QGroundControl with an IP link.
 
-To make this permanent, run the following command:
+## Companion Computer Setup
 
-```sh
-sudo sh -c "iptables-save > /etc/iptables.ipv4.nat"
-```
+General instructions for installing GStreamer and starting the stream on your companion computer are provided in the [QGroundControl VideoReceiver README](https://github.com/mavlink/qgroundcontrol/blob/master/src/VideoReceiver/README.md).
 
-Now edit the file /etc/network/interfaces and add the following line to the bottom of the file: 
-
-```sh
-up iptables-restore < /etc/iptables.ipv4.nat
-```
-
-# Gstreamer Installation
-
-To install gstreamer packages on the computer and on the Odroid C1 and start the stream, follow the instruction given in the [QGroundControl README](https://github.com/mavlink/qgroundcontrol/blob/master/src/VideoReceiver/README.md).
-
-If you cannot start the stream on the Odroid with the uvch264s plugin, you can also try to start it with the v4l2src plugin:
+If you cannot start the stream with the uvch264s plugin, you can also try to start it with the v4l2src plugin:
 
 ```sh
```

Comment:
I'd remove this as well and refer to the QGC README so we have highest chances of it being correct.

Reply:
@bkueng You know that means there is NOTHING left of the original doc right - except the link to the QGC materials?
Done.